lib/ohai/mixin/network_helper.rb

```diff
@@ -37,6 +38,11 @@ def hex_to_dec_netmask(netmask)
         dec
       end
 
+      def ip?(hostname)
```

Comment:
Addrinfo has it's own `ip?`, use that instead

Reply:
Per discussion... `Addrinfo#ip*?` methods aren't quite what we want.